onnxruntime/core/providers/cpu/nn/qlinearconv.cc

```diff
@@ -360,7 +361,12 @@ Status QLinearConv::Compute(OpKernelContext* context) const {
 
   // Replicate the logic from MlasGemmU8X8Schedule to control the number of
   // worker threads used for the convolution.
-  constexpr int32_t maximum_thread_count = 16;
+  int32_t maximum_thread_count;
+  if (CPUIDInfo::GetCPUIDInfo().IsHybrid()) {
+    maximum_thread_count = 64;
```

Comment:
Is it possible to get the max thread count from the mlas lib instead of replicating the value here? Possibly using something similar to MlasGetPreferredBufferAlignment?

Reply:
Did not want to use MLAS references in general code flow outside MLAS

Reply:
We already do this. See core/platform/utils.cc. Also, it'll help avoid checking for hybrid core again since the setting of the mlas max thread count is based on that check done before.

Reply:
Let's address this in a later PR.

Reply:
Sure